data/human/news.txt

```diff
@@ -328,6 +383,28 @@ news "hyperspace relay engineer"
 			"All of human space is linked into a single communications network by relay beacons. It's something that most people just take for granted."
 			"Hyperspace links don't just allow faster-than-light travel. They also allow near-instantaneous communication, if you have the right sort of relay on both ends."
 			"The hyperspace communications network is a marvel of public infrastructure, but most people just take it for granted."
+			"Hyperspace relays are so integral to society, that even pirates don't dare to touch them, out of fear of becoming the number one enemy of the Republic."
+			"Due to the nightmare that is Republic law, new captains are probitited from accessing all the relays at the very beginning. Instead, they have to visit every system individually to register themselves, or purchase a map for the area."
+
+
+
+news "hyperspace scientist"
+	location
+		government "Republic" "Free Worlds" "Syndicate" "Neutral" "Independent"
+		attributes "research" "deep"
+	name
+		word
+			"Hyperspace researcher"
+			"Hyperspace scientist"
+			"Hyperspace specialist"
+	message
+		word
+			"It's hard to believe that modern-day society relies on what is effectively magical space highways."
+			"Even centuries later, we still have around the same amount of understanding as to how the hyperspace lanes were formed as Charles Plot did. The only extra thing that we know is that apparently some other race the Quarg know is involved with them."
+			"The Quarg don't appear to even bother with hyperspace lanes. Their hyperdrives apparently teleport their ships from one system to another. If that's true, then they would be able to visit the systems not accessible by using a normal hyperdrive."
+			"Even if they aren't directly linked, your ship can still detect systems in a certain away from you. Not that it matters, though, since we still can't jump to those systems."
```

Comment:
```suggestion
			"Even if they aren't directly linked by hyperspace lanes, your ship can still detect systems near you. Not that it matters, though, since we still can't jump to those systems."
```